hydrolib/core/io/mdu/models.py

```diff
@@ -0,0 +1,396 @@
+    Maxitverticalforestersal = 0
+    Maxitverticalforestertem = 0
+    Jaorgsethu = 1
+    Turbulencemodel = 3
+    Turbulenceadvection = 3
+    AntiCreep = 0
+    Maxwaterleveldiff = 0
+    Maxvelocitydiff = 0
+    Epshu = 0.0001
+
+
+class Physics(IniBasedModel):
+    UnifFrictCoef = 0.023
+    UnifFrictType = 1
+    UnifFrictCoef1D = 0.023
+    UnifFrictCoefLin = 0
+    Umodlin = 0
+    Vicouv = 1
+    Dicouv = 1
+    Vicoww = 5e-05
+    Dicoww = 5e-05
+    Vicwminb = 0
+    Smagorinsky = 0
+    Elder = 0
+    Irov = 0
+    wall_ks = 0
+    Rhomean = 1000
+    Idensform = 2
+    Ag = 9.81
+    TidalForcing = 0
+    Doodsonstart = 55.565
+    Doodsonstop = 375.575
+    Doodsoneps = 0
+    Salinity = 0
+    InitialSalinity = 0
+    Sal0abovezlev = -999
+    DeltaSalinity = -999
+    Backgroundsalinity = 30
+    InitialTemperature = 6
+    Secchidepth = 2
+    Stanton = -1
+    Dalton = -1
+    Backgroundwatertemperature = 6
+    SecondaryFlow = 0
+    EffectSpiral = 0
+    BetaSpiral = 0
+    Temperature = 0
+
+
+class Wind(IniBasedModel):
+    ICdtyp = 2
+    Cdbreakpoints: List[float] = [0.00063, 0.00723]
+    Windspeedbreakpoints: List[int] = [0, 100]
+    Rhoair = 1.205
+    PavBnd = 0
+    PavIni = 0
+
+    _split_to_list = get_split_string_on_delimeter_validator(
+        "Cdbreakpoints", "Windspeedbreakpoints", delimiter=" "
+    )
+
+
+class Waves(IniBasedModel):
+    Wavemodelnr = 0
+    WaveNikuradse = 0.01
+    Rouwav = "FR84"
+    Gammax = 1
+
+
+class Time(IniBasedModel):
+    RefDate = 20200101
+    Tzone = 0
+    DtUser = 60
+    DtNodal: Optional[int] = None
+    DtMax = 10
+    DtInit = 1
+    Tunit = "S"
+    TStart = 0
+    TStop = 7200
+    AutoTimestepNoStruct = 1
+
+
+class Restart(IniBasedModel):
+    RestartFile: Optional[Path] = None
+    RestartDateTime: Optional[str] = None
+
+
+class Boundary(IniBasedModel):
+    quantity: str
+    nodeId: str
+    forcingfile: Optional[ForcingModel] = "FlowFM_boundaryconditions1d.bc"
+    bndWidth1D: float
+
+    def is_intermediate_link(self) -> bool:
+        return True
+
+
+class Lateral(IniBasedModel):
+    id: str
+    name: str
+    nodeId: str
+    discharge: str
+
+
+class ExtGeneral(INIGeneral):
+    fileVersion: str = "2.01"
+    fileType: Literal["extForce"] = "extForce"
+
+
+class ExtModel(INIModel):
+    general: ExtGeneral
+    boundary: List[Boundary]
+    lateral: List[Lateral]
+
+    _split_to_list = make_list_validator("boundary", "lateral")
+
+    @classmethod
+    def _ext(cls) -> str:
+        return ".ext"
+
+    @classmethod
+    def _filename(cls) -> str:
+        return "bnd"
+
+
+class ExternalForcing(IniBasedModel):
+    ExtForceFile: Optional[Path] = None
+    ExtForceFileNew: Optional[ExtModel] = "FlowFM_bnd.ext"
+
+    def is_intermediate_link(self) -> bool:
+        return True
+
+
+class Trachytopes(IniBasedModel):
+    TrtRou = "N"
+    TrtDef: Optional[int] = None
+    TrtL: Optional[int] = None
+    DtTrt = 60
+
+
+class Output(IniBasedModel):
```

Comment:
A lot of these values are booleans, but we are assigning them ints, is that we want want to do?

Reply:
Should be a followup to document all the fields a bit better and more complete. They're now based on the test files.